webrtc.html

```diff
@@ -4331,6 +4331,65 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <dt>RTCIceCandidate remote</dt>
           <dd><p>The remote ICE candidate.</p></dd>
         </dl>
+        
+        <h3>RTCIceTransportState Enum</h3>
+        <p><dfn>RTCIceTransportState</dfn> represents the current state of the ICE transport.</p>
+        <dl class="idl" title="enum RTCIceTransportState">
+        	<dt>new</dt>
+        	<dd>
+        		<p>
+        			The <code><a>RTCIceTransport</a></code> object is gathering candidates and/or waiting for
+        			remote candidates to be supplied.
+        		</p>
+        	</dd>
+        	<dt>checking</dt>
+        	<dd>
+        		<p>
+        			The <code><a>RTCIceTransport</a></code> has received at least one remote candidate,
+        			and is checking candidate pairs but has not yet found a successful candidate pair,
+        			or consent checks [[!RFC7675]] have failed on a previously successful candidate pair.
+        			In addition to checking, it may also still be gathering. 
+                        </p>
+                </dd>
+                <dt>connected</dt>
+                <dd>
+                        <p>
+                        	The <code><a>RTCIceTransport</a></code> has received found a usable connection
+                        	but is still checking other candidate pairs to see if there is a better connection.
+                        	It may also still be gathering and/or waiting for additional remote candidates.
+                        </p>
+                </dd>
+                <dt>completed</dt>
+                <dd>
+                        <p>
+                        	The <code><a>RTCIceTransport</a></code> has finished gathering, received an
+                        	indication that there are no more remote candidates, finished checking all
+                        	candidate pairs and has found a connection.
+                        </p>
+                </dd>
+                <dt>disconnected</dt>
+                <dd>
+                        <p>
+                        	The <code><a>RTCIceTransport</a></code> has not yet finished gathering, and has not yet received
+                        	an indication that there are no more remote candidates.  However, all candidate pairs tested thus
+                        	far have failed (or consent checks [[!RFC7675]] once successful, have now failed).  Since other
+                        	candidate pairs may become available for testing, the "failed" state has not been reached.
+                        </p>
+                </dd>
+                <dt>failed</dt>
+                <dd>
+                        <p>
+                        	The <code><a>RTCIceTransport</a></code> has finished gathering, received an indication that there are
+                        	no more remote candidates, finished checking all candidate pairs, and failed to find a connection.
```

Comment:
The descriptions for "disconnected" and "failed" read from the consent checks perspective as:
- if the current state is "connected" (but not yet "completed") and consent checks fail I should switch to to "disconnected"
- if the current state is "completed" and consent checks fail I should switch to....? I'm assuming "failed", as "completed" says gathering is finished, and "disconnected" explicitly includes that gathering has not finished yet. If failed is the desired state it would be good to add one more sentence about failed consent checks to be 100% clear.